viewer/src/dataModels/pointCloud/internal/PointCloudManager.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright 2020 Cognite AS
+ */
+
+import * as THREE from 'three';
+import { PointCloudFactory } from './PointCloudFactory';
+import { PointCloudMetadataRepository } from './PointCloudMetadataRepository';
+import { PotreeGroupWrapper } from './PotreeGroupWrapper';
+import { PotreeNodeWrapper } from './PotreeNodeWrapper';
+
+export class PointCloudManager<TModelIdentifier> {
```

Comment:
This looks unit testable ;)

Reply:
Yes, I want to add tests for all of these. The manager itself isn't that important. The important ones are the individual pieces. Like the factory etc.